app/src/main/java/com/example/scoutconcordia/Activities/MapsActivity.java

```diff
@@ -1333,6 +1366,9 @@ public void onMapClick(LatLng latLng) {
                 showAllMarkers();
                 resetGetDirectionParams();
 
+                from.setVisibility(View.INVISIBLE);
+                to.setVisibility(View.INVISIBLE);
+                travelTime.setVisibility(View.INVISIBLE);
                 //System.out.println(latLng);
```

Comment:
Will we need this in the future? If not we can remove it

Reply:
yes, in order to make the estimated time disappear after clicking on map we need it